webpack.config.js

```diff
@@ -0,0 +1,95 @@
+const path = require("path");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
+const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+  entry: {
+    main: "./src/frontend/index.ts",
+    editor: "./src/frontend/editor.ts",
+    pdf: "./src/frontend/pdf.ts"
```

Comment:
Thanks for your efforts with the PDF config with the extra fonts!
But when I try to export a PDF, the font is not recognized:
```
Mar 14 15:01:30 INTEGREAT CMS - ERROR: The file fonts/DejaVuSans.ttf?60fc56f7c4ca745a7664e9af5ac42ecb was not found in the static directories ['/home/timo/job/integreat/integreat-cms/src/../node_modules', '/home/timo/job/integreat/integreat-cms/src/cms/static', '/home/timo/job/integreat/integreat-cms/.venv/lib/python3.7/site-packages/django/contrib/admin/static', '/home/timo/job/integreat/integreat-cms/.venv/lib/python3.7/site-packages/filer/static', '/home/timo/job/integreat/integreat-cms/.venv/lib/python3.7/site-packages/mptt/static']
File '/static/fonts/DejaVuSans.ttf?60fc56f7c4ca745a7664e9af5ac42ecb' does not exist
```
Maybe this is because the PDF renderer doesn't recognize that the `?` does not belong to the real filename? Is there are way to ignore the version hash for the pdf module? Or do we have to revert the changes for the PDF font?

When I export a page after inserting an icon, I also get the following error, but this bug also exists on the develop branch, so I don't this it it part of this PR:
```
getFile '../../../../static/svg/world-wide-web.svg' None '/home/timo/job/integreat/integreat-cms'
Traceback (most recent call last):
  File "/home/timo/job/integreat/integreat-cms/.venv/lib/python3.7/site-packages/xhtml2pdf/context.py", line 773, in _getFileDeprecated
    nv = self.pathCallback(name, relative)
  File "/home/timo/job/integreat/integreat-cms/src/cms/utils/pdf_utils.py", line 145, in link_callback
    result = finders.find(uri)
  File "/home/timo/job/integreat/integreat-cms/.venv/lib/python3.7/site-packages/django/contrib/staticfiles/finders.py", line 268, in find
    result = finder.find(path, all=all)
  File "/home/timo/job/integreat/integreat-cms/.venv/lib/python3.7/site-packages/django/contrib/staticfiles/finders.py", line 103, in find
    matched_path = self.find_location(root, path, prefix)
  File "/home/timo/job/integreat/integreat-cms/.venv/lib/python3.7/site-packages/django/contrib/staticfiles/finders.py", line 120, in find_location
    path = safe_join(root, path)
  File "/home/timo/job/integreat/integreat-cms/.venv/lib/python3.7/site-packages/django/utils/_os.py", line 31, in safe_join
    'component ({})'.format(final_path, base_path))
django.core.exceptions.SuspiciousFileOperation: The joined path (/home/timo/job/static/svg/world-wide-web.svg) is located outside of the base path component (/home/timo/job/integreat/integreat-cms/node_modules)
Need a valid file name!
'<img src="../../../../static/svg/world-wide-web.svg" style="width: 15px; height: 15px;"/>'
```

Reply:
I thought it worked cause I got a PDF file but did not look into the console.
I changed the output format to have the hash in the actual file name.

Reply:
Thanks! The DejaVuSans font is mainly required for Arabic letters, for latin letters the output isn't that much different...